Mathlib/Analysis/Analytic/Basic.lean

```diff
@@ -373,26 +381,29 @@ variable {f g : E → F} {p pf pg : FormalMultilinearSeries 𝕜 E F} {x : E} {r
 /-- Given a function `f : E → F` and a formal multilinear series `p`, we say that `f` has `p` as
 a power series on the ball of radius `r > 0` around `x` if `f (x + y) = ∑' pₙ yⁿ` for all `‖y‖ < r`.
 -/
-structure HasFpowerSeriesOnBall (f : E → F) (p : FormalMultilinearSeries 𝕜 E F) (x : E) (r : ℝ≥0∞) :
-  Prop where
+structure HasFPowerSeriesOnBall (f : E → F) (p : FormalMultilinearSeries 𝕜 E F) (x : E) (r : ℝ≥0∞) :
+    Prop where
+  /-- The provided radius is at mots the `FormalMultilinearSeries.radius`. -/
```

Comment:
This docstring contained a typo. Adding docs is not needed during the port, and should be avoided for reasons like these...
```suggestion
```